### ISSUE TITLE
Change Namespace to Match Folder Structure Breaks Razor Using Clause

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#57885]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Change-Namespace-to-Match-Folder-Structu/1589588)._

---
This has happened each time I've done it.  Using the "Change namespace to match folder structure" updates all of the referencing using clauses but the @using declaration in razor files is broken, with spaces being added between the "@" symbol and "using".

Should be
@using My.New.Namespace

Instead, is:
@      using My.New.Namespace

---
#### Feedback Bot on 11/18/2021, 07:06 PM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

#### Feedback Bot on 11/19/2021, 09:43 AM: 

<p>Thank you for sharing your feedback! Our teams prioritize action on product issues with broad customer impact. See details at: <a target='_blank' href="https://docs.microsoft.com/en-us/visualstudio/ide/report-a-problem?view=vs-2019#faq">https://docs.microsoft.com/en-us/visualstudio/ide/report-a-problem?view=vs-2019#faq</a>. In case you need answers to common questions or need assisted support, be sure to use <a target='_blank' href="https://visualstudio.microsoft.com/vs/support/">https://visualstudio.microsoft.com/vs/support/</a>. We’ll keep you posted on any updates to this feedback.</p>

---
### Original Solutions
(no solutions)

**Comment by ryzngard** [comment]
This is specifically on the "Sync Namespaces" from solution explorer. Was able to reproduce locally with the provided MWE

**Comment by BrianLParker** [comment]
This is still happening..
![NamespaceRename](https://user-images.githubusercontent.com/8317299/161453712-a6e921c7-7b5c-48e0-aeb4-a6acbdcbcb18.gif)
.

**Comment by ryzngard** [comment]
Known issue, and should still be happening. The root cause is that C# formatting gets applied so code gets indented. 

```csharp
namespace MyNamespace
{
using SomethingInRazorFile; // <-- this is what generated code usually looks like
}
```

```
namespace MyNewNamespace
{
	using SomethingInRazorFile; // <-- this is what it looks like after applying the code fix.
}
```

Unfortunately that also means it's potentially complicated to fix and hasn't been prioritized over other things yet.

**Comment by sloncho** [comment]
This is still an issue - and essentially kills any effort to refactor a huge projects

**Comment by ryzngard** [comment]
This should be fixed in 17.13 preview 2
